frontend/src/app/output/page-content/open-api-model/open-api-model.component.ts

```diff
@@ -18,7 +18,9 @@ export class OpenApiModelComponent implements OnInit {
 
   ngOnInit() {
     this.openApiRows = this.openApiModule.openApiRows;
-    this.notificationService.showError(`Error while loading Model`, this.openApiModule.errors[0]);
+    if(this.openApiModule.errors[0] !== ' ' && this.openApiModule.errors.length !== 0) {
```

Comment:
Conditions should be reverted. First check lenght !== 0, and then check content of 1st item